examples/webbilling-demo/src/util/PurchasesLoader.ts

```diff
@@ -37,7 +37,11 @@ const loadPurchases: LoaderFunction<IPurchasesLoaderData> = async ({
       Purchases.configure(
         apiKey,
         appUserId,
-        {},
+        {
+          additionalHeaders: {
+            "X-Rc-Canary": "noemail",
+          },
+        },
```

Comment:
This was not intentional, right?
```suggestion
        {},
```

Reply:
Thanks, it got late 😓